hs_core/templates/search/search.html

```diff
@@ -0,0 +1,204 @@
+                                            {% endif %}>{{ item.1 }}
+                                        </span>
+                                        {#<a href="{{ request.get_full_path }}&amp;selected_facets={{ key }}_exact:{{ item.0|urlencode }}">{{ item.0 }}</a>#}
+                                    </li>
+                                {% endfor %}
+                                </ul>
+                            </div>
+                        </div>
+                    </div>
+                {% endif %}
+            {% endfor %}
+        </div>
+
+        <div class="col-lg-9" id="items">
+            <div class=resource-search>
+                <form method="get" action=".">
+                    {% for field in form %}
+                     <div class="fieldWrapper">
+                         {{ field }}
+                    </div>
+                {% endfor %}
+                </form>
+            </div>
+            <table class="table table-striped table-hover" id="item-selectors">
+                <tr>
+                    <th>Title</th>
+                    <th>Type</th>
+                    <th>First Author</th>
+                    <th>Last Modified</th>
+                </tr>
+                {% if query %}
+                    {% for result in page.object_list %}
+                        <tr>
+                            <td><a href="{{ result.object.get_absolute_url }}">{{ result.object.title }}</a></td>
+                            <td>{{ result.object|resource_type }}</td>
+                            {% if result.object.first_creator.description %}
+                            <td><a href="{{ result.object.first_creator.description }}">{{ result.object.first_creator.name  }}</a></td>
+                            {% else %}
+                                <td>{{ result.object.first_creator.name }}</td>
+                            {% endif %}
+                            <td>{{ result.object.updated|date }} {{ result.object.updated|time }}</td>
+                        </tr>
+                    {% empty %}
+                    <p>No results found.</p>
+                    {% endfor %}
+                {% endif %}
+            </table>
+        </div>
+    </div>
+</div>
+{% endblock %}
+
+
+{% block extra_js %}
+    <script type="text/javascript">
+    var reorderDivs = function(){
+        var faceted_fields = ['author', 'subjects', 'resource_type', 'public', 'creators', 'discoverable', 'language'];
+         var div_ordering = [];
+         faceted_fields.forEach(function(field){
+             var faceting_div = "faceting-"+field;
+             div_ordering.push(faceting_div);
+         });
+         var i;
+         for (i = 1; i < div_ordering.length; i++) {
+             var div0 = "#" + div_ordering[i-1];
+             var div1 = "#" + div_ordering[i];
+             $(div1).insertAfter(div0);
+         }
+    }
+
+    var clearAllFacted = function(){
+        window.localStorage.clear();
```

Comment:
See comment about not using `localStorage.clear()` below.

Also you can just use `localStorage` there is no need to qualify it with `window.localStorage`